### PR TITLE
Toggle the default status of a few defines to avoid confusing new people

### PR DESCRIPTION
**Pull request by arthurtilly**
Some of these might be a bit contentious so I welcome discussion

### CODE REVIEW

File: include/config/config_objects.h [hunk 1]
```diff
@@ -27,7 +27,7 @@
 /**
  * Allow previewing blue coin locations from a blue coin switch when Mario is standing on the blue coin switch.
  */
-// #define BLUE_COIN_SWITCH_PREVIEW
+#define BLUE_COIN_SWITCH_PREVIEW
```
Comment (gheskett) [inline include/config/config_objects.h:30]: This one feels like a bit of a wild card to enable by default compared to everything else here

Comment (arthurtilly) [reply include/config/config_objects.h:30]: thats fair i can disable it

**Comment by gheskett** [comment]
Moving to 2.2, since this could very easily catch people by surprise

**Review by gheskett — CHANGES_REQUESTED** [review]
Nope nevermind, this branch needs to be rebased for 2.2

**Comment by arthurtilly** [comment]
should be rebased now

**Review by gheskett — APPROVED** [review]

**Review by someone2639 — APPROVED** [review]
Changes look and play good, though i wonder what was weird about the IA8 coins

**Comment by arthurtilly** [comment]
the ia8 coins are just very noticeably not vanilla whenever starting a hackersm64 repo

**Comment by someone2639** [comment]
I guess they are jarring but idk if they have that same effect after the first time

**Pull request merged**